integration/azure_auth.go

```diff
@@ -0,0 +1,20 @@
+package integration
+
+import (
+	"encoding/json"
+	"os"
+)
+
+type azureCredentialsFile struct {
+	ClientSecret string `json:"keyId"`
```

Comment:
Are these mapped incorrectly? Shouldn't keyId map to ClientID and secret map to ClientSecret?